src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/controller/WelcomeControllerIntegrationTest.java

```diff
@@ -125,93 +146,439 @@ public void welcomeApiTest() throws Exception {
 
 
     @Test
-     public void createOrgRoleMappingTest() throws Exception {
+    public void createOrgRoleMappingTest() throws Exception {
+        Mockito.when(crdFeignClientFallback.createRoleAssignment(any()))
+                .thenReturn(new ResponseEntity<>(IntTestDataBuilder
+                        .buildListOfUserProfiles(true, false,"1", "2", ROLE_NAME_STCW, ROLE_NAME_TCW,
+                                true, true, false,
+                                true,"BFA1", "BFA2",false), HttpStatus.OK));
+
+        UserRequest request = UserRequest.builder()
+                .users(Arrays.asList("21334a2b-79ce-44eb-9168-2d49a744be9z", "21334a2b-79ce-44eb-9168-2d49a744be9x"))
+                .build();
+        logger.info(" createOrgRoleMappingTest...");
+        String uri = "/am/role-mapping/staff/users";
+        setRoleAssignmentWireMock(HttpStatus.CREATED, SAMPLE_RAS_RESPONSE);
+
+        mockMvc.perform(post(uri)
+                .contentType(JSON_CONTENT_TYPE)
+                .headers(getHttpHeaders())
+                .content(mapper.writeValueAsBytes(request)))
+                .andExpect(status().is(200))
+                .andReturn();
```

Comment:
We need to put some asserts for following 
1. What is the actual status of request Approved /Rejected
2. How many assignments records created- 1/2 
3. Status of each assignment records.